script/create-build-settings.js

```diff
@@ -9,6 +9,10 @@ function getBuildSettings(options) {
   return {
     type: options.buildtype,
     brandConsolidationEnabled: !!options['brand-consolidation-enabled'],
+    metrics: {
+      enabled: !!options['frontend-metrics-enabled'],
+      samplePercentage: options['metrics-sample-percentage'],
+    },
```

Comment:
I'm not against having these here, but unless they're possibly going to be changed outside of our regular build cycles, it might be easier to have them closer to the rest of the metrics code.

Reply:
I could see us wanting to turn down the sample percentage outside of build cycle, but I'm not sure about turning it on or off.

Reply:
I removed the build flag and left the sampling percentage until after we have a sync this afternoon (9/26) to discuss our business case a bit more in-depth. Afterwords we'll have a better idea if this needs to be a configurable setting or hardcoded constant.